src/services/services.ts

```diff
@@ -471,9 +481,13 @@ namespace ts {
             return this.checker.getReturnTypeOfSignature(this);
         }
 
-        getDocumentationComment(): SymbolDisplayPart[] {
+        getDocumentationComment(typeChecker?: TypeChecker): SymbolDisplayPart[] {
             if (this.documentationComment === undefined) {
                 this.documentationComment = this.declaration ? JsDoc.getJsDocCommentsFromDeclarations([this.declaration]) : [];
+
+                if (this.declaration && this.documentationComment.length === 0 && hasJSDocInheritDocTag(this) && typeChecker) {

```

Comment:
This might whole block might be neater as `if (this.declaration) { ... } else { this.documentationComment = []; }`